malaria-app-android/src/main/java/com/peacecorps/malaria/code/activities/NewHomeActivity.java

```diff
@@ -63,13 +64,13 @@ public void onClick(View v) {
                 finish();
             }
         });
-        userProfile.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                startActivity(new Intent(getApplicationContext(), UserProfile.class));
-                finish();
-            }
-        });
+//        userProfile.setOnClickListener(new View.OnClickListener() {
```

Comment:
Remove these lines

Reply:
Initially, code had buttons in every layout for navigation instead of bottom navigation so listener was available in every file. I still have to send PR for other items in bottom navigation except "User Profile", will it be okay if I do all java changes when shifting this class file to MVP?